transport-native-io_uring/src/main/c/netty_io_uring_native.c

```diff
@@ -263,9 +263,8 @@ static jlongArray netty_io_uring_setup(JNIEnv *env, jclass clazz, jint entries)
     struct io_uring_params p;
     memset(&p, 0, sizeof(p));
 
-#ifdef IORING_SETUP_SUBMIT_ALL
+    // Always use IORING_SETUP_SUBMIT_ALL if possible.
     p.flags = IORING_SETUP_SUBMIT_ALL;
```

Comment:
What kernel introduced this? Just making sure we don't risk getting EINVAL on kernels we intend to support.

Reply:
5.18... hmm maybe you are right and we should be a bit more careful.

Reply:
It was added in 5.18 according to https://man.archlinux.org/man/io_uring_setup.2.en#IORING_SETUP_SUBMIT_ALL
But we claim to support at least 5.14.

Reply:
yeah... let me do some dance to detect if its supported or not and only if it is supported adding it.